tictactoe.py

```diff
@@ -32,12 +41,28 @@ def draw_board(self):
         print(f"| {self.board[2][0]} | {self.board[2][1]} | {self.board[2][2]} |")
         print(divider)
 
+    def make_move(self, player, move):
+        self.board[(move - 1) // 3][(move - 1) % 3] = self.markers[player]
+
+    def play_game(self):
+        for player in self.players:
+            player_move = input(f"{player.name}, please enter move: ")
+            self.make_move(player, int(player_move))
+            self.draw_board()
+
 
 class Player:
-    def __init__(self, name):
-        self.name = name
+    def __init__(self, player_info):
+        self.name = player_info[0]
+        self.type = player_info[1]
 
 
 if __name__ == "__main__":
-    Info.welcome_message()
-    Info.game_instructions()
+    UserInterface.welcome_message()
+    UserInterface.game_instructions()
+    game = TicTacToe()
+    player_list = UserInterface.get_player_info()
+    game.create_players(player_list)
```

Comment:
Is there a use case where we would _not_ want to run `game.assign_player()` immediately after `game.create_players()`? If not, does this add complexity to our API?

Reply:
Created a new method called set_up_players that gets the player information and then assigns it. The seperate create and assign methods still exist to keep things seperate, but they are both called by set_up_players.